nix/default.nix

```
@@ -0,0 +1,10 @@
+{
+    config ? {},
+    pkgs ? import ./pkgs.nix { inherit config; }
+}:
+ let
+  # put all main targets and shells together for easy import
+  shells = pkgs.callPackage ./shell.nix { };
+in {
+  inherit pkgs shells;
```

Actually you could avoid the `let`/`in` block entirely by doing this:
```suggestion
  inherit pkgs;
  shells = pkgs.callPackage ./shell.nix { };
```